Giveaways.md

```diff
@@ -0,0 +1,15 @@
+# Giveaways
+From time to time Devstaff has the opportunity to perform some giveaways during the meetups. The prizes are provided by [sponsors](https://github.com/devstaff-crete/DevStaff-Heraklion/blob/master/Sponsors.md) in return of advertising.
+
+Here we describe the process, the tools and the rules we use for these giveaways.
```

Comment:
Please make the whole sentense as follows:
`The process, the tools and the rules we use for these giveaways are described here:`.